mlflow/R/mlflow/R/tracking-rest.R

```diff
@@ -20,6 +20,7 @@ try_parse_response_as_text <- function(response) {
 }
 
 #' @importFrom base64enc base64encode
+#' @importFrom utils packageVersion
```

Comment:
Could we qualify the `packageVersion()` call i.e. `utils::packageVersion()` instead of importing?